Staging (#138)
makerdao/endgame-docs: maker-core/cvc.md

# Aligned Voter Committees (AVC)

> Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.

Reviewer: Minor comment that the AVC only needs one MKR holder, so it isn't a group of MKR holders.

> Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.

Author: @astronautthis I believe all members must hold MKR. I asked Patrick, will let you know.

> Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.

Author: What do you think about:

Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared strategic vision for the Maker protocol. They have two main tasks in governance:
1. To ensure that all [Aligned Delegates (ADs)](delegates.md) that are affiliated with them, execute the AVC's strategic vision when voting on governance issues.   
2. To propose modifications to the Scope Artifacts to strengthen (solidify? ossify?) them over time.

ty

> Aligned Voter Committees (AVCs) are organized groups of MKR holders who have a shared vision for the protocol. They produce and publish documentation communicating their vision and strategy which is used by Aligned Delegates to direct the evolution of the Maker Protocol.

Reviewer 2: Presumably non-Aligned VCs also share a strategic vision?

{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}

## Overview

> Each AVC produces Aligned Governance Strategy and Aligned Scope Artifact documents to update the [Scope Artifacts](list-of-scopes.md). This set of documents provides general guidelines for how [Aligned Delegates (ADs)](delegates.md) that follow their governance strategy should vote.

Reviewer: ```suggestion
Each AVC produces an Aligned Governance Strategy document and Aligned Scope Artifact documents to update the Scope Artifacts of each [Scope]](list-of-scopes.md) once every quarter. The Aligned Governance Strategy document provides general guidelines for how Aligned Delegates (ADs) that follow their governance strategy should vote.
```

> The documents produced by the AVCs abstract away the minutiae of day-to-day governance. Since Aligned Delegates execute AVC strategies, they allow MKR Holders to be reasonably informed and secure with regard to the general direction of the chosen delegate.

Reviewer 2: They = the documents, ADs, AVCs?

According to their own strategy, AVCs fall somewhere in the spectrum that goes from **Hawk** to **Dove**:
  - Hawkish AVCs focus more on short-term income and asset accumulation, tending to prefer higher spreads and lower budgets
  - Dovish AVCs focus more on long-term growth and innovation, tending to prefer lower spreads and higher budgets
  - Balanced AVCs try to find a middle ground between the two

> The Hawk, Dove, Balanced is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less Balanced.

Reviewer 2: ```suggestion
The Hawk/Balanced/Dove is a continuous spectrum, so many different hawkish or dovish AVCs can exist that are more or less Balanced.
```

## Eligibility Requirements
In order for a voter committee to be considered an AVC, it must have at least one member who is an [Alignment Conserver (AC)](constitutional-conservers.md) and a verified MKR holder.

To become Active AVCs, AVCs are subject to additional eligibility requirements. For each quarterly governance cycle they must:
   - Create one Aligned Governance Strategy document and five Aligned Scope Proposals
   - Hold two AVC Subcommittee meetings for each scope

> The [Governance Scope](https://mips.makerdao.com/mips/details/MIP113) recognizes Active AVCs. It can revoke this recognition anytime the AVC fails to meet all eligibility requirements.

Reviewer 2: (Not sure what "recognizes" means here)

Author: https://mips.makerdao.com/mips/details/MIP101#2-5-3-aligned-voter-committee-recognition-process

## Member Participation Rewards

> Member Participation Rewards is a compensation scheme available to AVC members. It considers their participation in each scope.

Reviewer 2: ```suggestion
Member Participation Rewards is a compensation scheme available to AVC members. Rewards are distributed per scope.
```

To be eligible for compensation, AVC members must:

>  - Attend the two AVC Subcommittee Meetings

Reviewer 2: ```suggestion
 - Attend the two quarterly AVC Subcommittee Meetings for a given scope
```
